ci-operator/step-registry/openshift/e2e/test/openshift-e2e-test-commands.sh

```diff
@@ -417,6 +417,45 @@ do
 done
 echo "$(date) - all imagestreams are imported."
 
+echo "Collecting cluster data for analysis..."
```

Comment:
Thanks Deep!  Definitely going to want to get feedback from Stephen and possible Devan.  For my own curiosity, is this anything we could do as a test (inside the e2e test execution itself)?   Not saying we should just curious if that is an alternative and if so comparing the approaches.

Reply:
I suppose what I am wondering is if we could just put the calls to get those invariant values in

https://github.com/openshift/origin/blob/748941c870eb4dce2ab6b5c671ababf32a78bb2c/pkg/riskanalysis/write_test_failure_summary.go#L18

We will have to do work in that area one way or another  and if it is less intrusive / more straight forward we might consider it